strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/ActuatorEndpointControllerTest.java

```diff
@@ -23,6 +26,13 @@
 public class ActuatorEndpointControllerTest
         extends RestAssuredBaseTest
 {
+    private final String UNAUTHORIZED_MESSAGE = "Full authentication is required to access this resource";
```

Comment:
This will cause #1466 and #1438 to popup again. Can we create a method in `RestAssuredBaseTest` which we could use to retrieve the message and re-use it in the tests? If so, could you please create an issue and label it with #hacktoberfest? Thanks in advance! :)

Reply:
I've done it myself in this PR so it won't pop, thanks